packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -30,6 +30,7 @@ import { isBarrierSupported, removeBarrier } from '../SmartChart/Helpers/barrier
 import BaseStore from '../../base-store';
 
 const store_name = 'trade_store';
+const fallback_symbol = 'frxAUDJPY';
```

Comment:
What if the fallback_symbol(`frxAUDJPY`) is not available? 
Can we set it to first available symbol or first available forex symbol based on `contracts_for` response?

Reply:
Yep. We now check if a `symbol` is offered dynamically, based on API response.